presto-iceberg/src/test/java/io/prestosql/plugin/iceberg/AbstractTestIcebergSmoke.java

```diff
@@ -699,6 +726,12 @@ public void testDayTransformTimestamp()
                     "(DATE '2015-05-15', 2, NULL, NULL, 4, 5), " +
                     "(DATE '2020-02-21', 2, NULL, NULL, 6, 7)";
         }
+        if (!columnStatisticsCollected) {
```

Comment:
I think these tests will become simpler if we make it similar to how we handle ORC. For example instead of defining `columnStatisticsCollected` and adding new methods and sublcassing. Could we not just do 
if (format == AVRO) and test appropriately?